src/sentry/web/api.py

```diff
@@ -1,80 +1,23 @@
 from __future__ import absolute_import, print_function
 
 import base64
-import math
 
-import io
-import jsonschema
 import logging
-import random
-import six
-import traceback
-import uuid
-
-from time import time
 
 from django.conf import settings
-from django.contrib.auth.models import AnonymousUser
-from django.core.cache import cache
-from django.core.urlresolvers import reverse
-from django.core.files import uploadhandler
-from django.http import HttpResponse, HttpResponseRedirect, HttpResponseNotAllowed
-from django.http.multipartparser import MultiPartParser
-from django.utils.encoding import force_bytes
-from django.views.decorators.cache import never_cache, cache_control
-from django.views.decorators.csrf import csrf_exempt
+from django.http import HttpResponse
+from django.views.decorators.cache import cache_control
 from django.views.generic.base import View as BaseView
 from functools import wraps
-from querystring_parser import parser
-from symbolic import ProcessMinidumpError, Unreal4Crash, Unreal4Error
-from sentry_relay import ProcessingErrorInvalidTransaction
 
-from sentry import features, options, quotas
-from sentry.attachments import CachedAttachment
-from sentry.constants import DataCategory, ObjectStatus
-from sentry.coreapi import (
-    Auth,
-    APIError,
-    APIForbidden,
-    APIRateLimited,
-    ClientApiHelper,
-    ClientAuthHelper,
-    SecurityAuthHelper,
-    MinidumpAuthHelper,
-    safely_load_json_string,
-    logger as api_logger,
-)
-from sentry.event_manager import EventManager
 from sentry.interfaces import schemas
-from sentry.interfaces.base import get_interface
-from sentry.lang.native.unreal import (
-    merge_unreal_user,
-    unreal_attachment_type,
-    merge_unreal_context_event,
-    merge_unreal_logs_event,
-    write_applecrashreport_placeholder,
-)
 
-from sentry.lang.native.minidump import (
-    merge_attached_event,
-    merge_attached_breadcrumbs,
-    write_minidump_placeholder,
-    MINIDUMP_ATTACHMENT_TYPE,
-)
-from sentry.models import Project, File, EventAttachment, Organization
-from sentry.signals import event_accepted, event_received
-from sentry.quotas.base import RateLimit
-from sentry.utils import json, metrics
-from sentry.utils.data_filters import FilterStatKeys
-from sentry.utils.http import is_valid_origin, get_origins, is_same_domain, origin_from_request
-from sentry.utils.outcomes import Outcome, track_outcome
+from sentry.models import Project
+from sentry.utils import json
+from sentry.utils.http import get_origins, origin_from_request
 from sentry.utils.pubsub import QueuedPublisherService, KafkaPublisher
-from sentry.utils.safe import safe_execute
-from sentry.utils.sdk import configure_scope
 from sentry.web.helpers import render_to_response
 from sentry.web.client_config import get_client_config
-from sentry.relay.config import get_project_config
-from sentry.datascrubbing import scrub_data
```

Comment:
Collecting a few comments here:
- Can we move `allow_cors_options` to `api/base.py`? 
- Is the `SchemaView` documented anywhere and/or used? If not, I believe we can get rid of it, and everything that it needs (also update `urls.py` then).

Reply:
Getting rid of schemaview, but interfaces/schema cleanup is more complicated and probably needs new metrics to figure out.

Reply:
Yeah, I'd keep the schema for now, and also all the interfaces (other than `from_raw` in security interfaces which we know are dead).